File: backend/app/Models/Hero/Hero.php
<?php

namespace App\Models\Hero;

use Eloquent;
use Illuminate\Database\Eloquent\Builder;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\HasMany;

/**
 * @mixin Eloquent
 * @mixin Builder
 */
class Hero extends Model
{
  protected $fillable = [
    'name',
    'sex',
    'description',
    'photo_url',
    'health',
    'damage_per_sec',
    'bullets_amount',
    'move_speed',
    'weapon_damage',
  ];
  protected $casts = [
    'damage_per_sec' => 'float', // или 'decimal:2'
    'move_speed' => 'float',     // или 'decimal:2'
    'weapon_damage' => 'float',  // или 'decimal:2'
  ];

  public function abilities(): HasMany
  {
    return $this->hasMany(Ability::class);
  }
}

File: UserPanel/Models/Hero.cs
using System.Text.Json.Serialization;
namespace UserPanel.Models;
public class Hero
{
    [JsonPropertyName("id")] public required int Id { get; set; }
    [JsonPropertyName("name")] public required string Name { get; set; }
    [JsonPropertyName("sex")] public required int Sex { get; set; }
    [JsonPropertyName("description")] public string? Description { get; set; }
    [JsonPropertyName("photo_url")] public required string PhotoUrl { get; set; }
    [JsonPropertyName("health")] public required int Health { get; set; }
    [JsonPropertyName("damage_per_sec")] public required decimal DamagePerSec { get; set; }
    [JsonPropertyName("bullets_amount")] public required int BulletsAmount { get; set; }
    [JsonPropertyName("move_speed")] public required decimal MoveSpeed { get; set; }
    [JsonPropertyName("weapon_damage")] public required decimal WeaponDamage { get; set; }
    [JsonPropertyName("abilities")] public List<Ability>? Abilities { get; set; }
}